tests/test_coverage_gaps.py

```
@@ -952,7 +952,13 @@ def test_sync_disconnect_returns_awaitable(self):
 
         iface = MagicMock()
         iface._exit_handler = None
-        iface.disconnect.return_value = MagicMock()
+
+        def _disconnect_no_timeout(*args, **kwargs):
```

Ruff (0.15.11)

[warning] Missing return type annotation for private function `_disconnect_no_timeout`

(ANN202)